__tests__/suggestedActions.js

```diff
@@ -9,6 +9,20 @@ function sleep(ms = 1000) {
 // https://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebDriver.html
 
 describe('suggested-actions command', async () => {
+    test('should show correctly formatted buttons when suggested actions are displayed', async() => {
```

Comment:
Can you change it from 4-space to 2-space? You find my VSCode extension to do the job.